### DISCUSSION TITLE
How to get the Prosys Severtime in opcua client?

### DISCUSSION THREAD

**Discussion by georgethomas08** [FreeOpcUa/python-opcua#1318] [Q&A]
Hi,
I need an help to retrieve prosys simulation server time in opcua client (python program)

**Comment by AndreasHeine** [comment]
just read it its a node under Server -> ServerStatus -> currentTime !?

**Reply by georgethomas08** [reply]
Can you help me with a syntax ? I need to read the time which is showing on Prosys simulation server

**Reply by AndreasHeine** [reply]
you should lookup the examples to get an idea how to do it! 
hint: node.get_value()
